index.html

```diff
@@ -4,195 +4,280 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
     <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,700;1,500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" media="screen and (max-width: 600px)" href="style.css">
+    <link rel="stylesheet" media="screen and (min-width: 601px)" href="desktop.css">
+    <!-- fontawesome -->
     <script src="https://kit.fontawesome.com/97d9fb687f.js" crossorigin="anonymous"></script>
-    <link rel="stylesheet" media="screen and (max-width: 765px)" href="style.css">
-    <script src='https://kit.fontawesome.com/a076d05399.js' crossorigin='anonymous'></script>
-    <title>Ayomide's portfolio</title>
+    <title>Ayomide S.E.P</title>
 </head>
 <body>
-    <section>
-        <header class="header">
-            <nav class="navbar">
-                <div class="logo">
-                    <img src="./assets/My Logo.png" alt="">
-                </div>
-                <div class="hamburger">
-                    <span class="line-top"></span>
-                    <span class="line-middle"></span>
-                    <span class="line-bottom"></span>
+    <header class="header">
+        <nav class="nav pad-aside bg-mobile ">
+            <a href="#" class="logo">
+                <img src="./assets/Logo.png" alt="fir">
+            </a>
+            <a href="#"  class="hamburger-container" ><img src="./assets/hamburger.png" class="hamburger" alt="se"></a>
+
+            <div class="mobile-menu hide ">
+                <ul>
+                    <li><a href="#portfolio" class="mobile-menu-link">Portfolio</a></li>
+                    <li><a href="#about" class="mobile-menu-link">About</a></li>
+                    <li><a href="#contact" class="mobile-menu-link">Contact</a></li>
+                </ul>
+                <div>
+                    <img src="./assets2/Normal Button/Tertiary/Icons/Icon.png" class="cancel-mobile" alt="">
                 </div>
-            </nav>
-            <div class="first intro">
-                <h1 class="title">
-                    I'm Ayomide.s.e.p<br>
-                    Glad to see you!
+            </div>
+            <div class="nav-link-desktop hide">
+                <ul>
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </div>
+        </nav>
```

Comment:
- [ ] kindly consider not displaying nav-items of your mobile version on the desktop version to make your work professional and follow the design provided